src/api/server/common.rst

```diff
@@ -208,6 +208,115 @@
            "locations"
         ]
 
+.. _api/server/dbs_info:
+
+==============
+``/_dbs_info``
+==============
+
+.. http:post:: /_dbs_info
+    :synopsis: Returns information of a list of the specified databases
+
+    Returns information of a list of the specified databases in the CouchDB
+    instance. This enables you to request information about multiple databases
+    in a single request, in place of multiple :get:`/{db}` requests.
+
+    :<header Accept: - :mimetype:`application/json`
+    :>header Content-Type: - :mimetype:`application/json`
+    :code 200: Request completed successfully
```

Comment:
What are the accepted query parameters?  I gather `keys` takes an array of database names? Are there other supported params to include here?

Reply:
Oh of course.  I guess I meant to ask about the keys in the body of the request. Would it be worth mentioning those here?  Perhaps `keys` is the only one.

Reply:
Yes, `keys` is only supported for now. I added one line for keys in 5679e53. Thanks again